Docs/Book/C++Examples/example13.cpp

```diff
@@ -0,0 +1,33 @@
+//
+// Generating depictions - example12.cpp
```

Comment:
Doh! I skipped the 'Preserving Molecules' section as I didn't think it was possible in C++. When I discovered that it can be done, I had to make a new example12.cpp and shunt everything down by one, which I clearly didn't do thoroughly enough.